docs/feature_space_cadet_shift.md

```diff
@@ -30,8 +36,17 @@ Also, by default, the Space Cadet applies modifiers LSPO_MOD and RSPC_MOD to key
 |Define                        |Default      |Description                                                                     |
 |------------------------------|-------------|--------------------------------------------------------------------------------|
 |`LSPO_KEY`                    |`KC_9`       |The keycode to send when Left Shift is tapped                                   |
-|`RSPC_KEY`                    |`KC_0`       |The keycode to send when Right Shift is tapped                                  |
+|`RSPC_KEY`                    |`KC_0`       |The modifier to apply to LSPO_KEY                                               |
 |`LSPO_MOD`                    |`KC_LSFT`    |The keycode to send when Left Shift is tapped                                   |
-|`RSPC_MOD`                    |`KC_RSFT`    |The keycode to send when Right Shift is tapped                                  |
+|`RSPC_MOD`                    |`KC_RSFT`    |The modifier to apply to RSPC_KEY                                               |
+|`LCPO_KEY`                    |`KC_9`       |The keycode to send when Left Ctrl is tapped                                    |
+|`RCPC_KEY`                    |`KC_0`       |The modifier to apply to LCPO_KEY                                               |
+|`LCPO_MOD`                    |`KC_LCTL`    |The keycode to send when Left Ctrl is tapped                                    |
+|`RCPC_MOD`                    |`KC_RCTL`    |The modifier to apply to RCPC_KEY                                               |
+|`LAPO_KEY`                    |`KC_9`       |The keycode to send when Left Alt is tapped                                     |
+|`RAPC_KEY`                    |`KC_0`       |The modifier to apply to LAPO_KEY                                               |
+|`LAPO_MOD`                    |`KC_LALT`    |The keycode to send when Left Alt is tapped                                     |
+|`RAPC_MOD`                    |`KC_RALT`    |The modifier to apply to RAPC_KEY                                               |
 |`DISABLE_SPACE_CADET_ROLLOVER`|*Not defined*|If defined, use the opposite Shift key to cancel Space Cadet                    |
-|`DISABLE_SPACE_CADET_MODIFIER`|*Not defined*|If defined, prevent the Space Cadet to apply a modifier to LSPO_KEY and RSPC_KEY|
+|`DISABLE_SPACE_CADET_MODIFIER`|*Not defined*|If defined, prevent the Space Cadet to apply a modifier to LSPO_
+KEY and RSPC_KEY|
```

Comment:
You accidentally a newline ;) And maybe change the wording here to "prevent Space Cadet applying a modifier"